cmake/develop_dependencies.cmake

```diff
@@ -11,11 +11,12 @@ FetchContent_Declare( ENDFtk
     GIT_SHALLOW     TRUE
     )
 
-FetchContent_Declare( eigen-adapter
-    GIT_REPOSITORY  https://github.com/njoy/eigen-adapter
-    GIT_TAG         origin/master
+FetchContent_Declare( eigen
```

Comment:
This makes me very happy.

Reply:
It did take a lot of testing to get it to work, isn't that right @nathangibson14 ?